test/demo/index.tsx

```diff
@@ -130,6 +144,44 @@ const Page: FC<{
         </ul>
       </>
     )
+  } else if (router === 'list 1') {
+    content = (
+      <>
+        <h2>Ignore unfocused buttons in a list</h2>
+        <ul>
+          <li data-keyux-ignore-hotkeys tabIndex={0}>
+            <button aria-keyshortcuts="v">First button</button>
+          </li>
+          <li data-keyux-ignore-hotkeys tabIndex={0}>
+            <button aria-keyshortcuts="v">Second button</button>
+          </li>
+        </ul>
+        <button aria-keyshortcuts="v">Outside button</button>
+      </>
+    )
+  } else if (router === 'list 2') {
+    content = (
+      <>
+        <h2>Click on panel element if focused on element with "data-keyux-hotkeys"</h2>
+        <ul>
+          <li
+            data-keyux-hotkeys="panel"
+            data-keyux-ignore-hotkeys
+            tabIndex={0}
+          >
+            <button aria-keyshortcuts="v">First button</button>
+          </li>
+          <li data-keyux-ignore-hotkeys tabIndex={0}>
```

Comment:
Let’ use the panel from both items to make example more real

Reply:
done